docker/intel-dgpu-driver.Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG I915_RELEASE=I915_23WW51.5_682.48_23.6.42_230425.56
 ARG FIRMWARE_RELEASE=23WW49.5_682.48
 
 # Intel Data Center GPU Driver for OpenShift version.
-ARG DRIVER_VERSION=2.1.0
+ARG DRIVER_VERSION=2.1.1
```

Comment:
according to https://github.com/intel/intel-data-center-gpu-driver-for-openshift/tree/main/release#driverversion
Since we will add PVC card support in this new release. I think 2.2.0 is better.
Or maybe we can upgrade to 3.0.0, if we think PVC card support is "the major feature improvement in the driver"

@hershpa what are your opinions on this?

Thanks!

Reply:
We could make the case for 2.1.1, 2.2.0 and 3.0.0. I think the 2.1.1 or 2.2.0 make the most sense. What's new is we are including the PVC FW binaries resulting in added support for PVC.

Reply:
@chaitanya1731  Then, how about changing it to 2.2.0?
Thanks! :-)

Reply:
As per our [definition](https://github.com/intel/intel-data-center-gpu-driver-for-openshift/tree/main/release#driverversion) we will only increment `y` when there are fixes or improvements in the driver from the [Intel GPU repository](https://github.com/intel-gpu/intel-gpu-i915-backports). So `2.2.0` is not ideal in this situation as we are not upgrading to the new release from the Intel GPU repository. However, I am fine with either `3.0.0` or `2.1.1` as we are simply adding the PVC support to current drivers. This is not an architecture change but a big feature enhancement. Any thoughts @uMartinXu @hershpa

Reply:
Another plus with 2.1.1 is we already have the driver container tested and ready to publish.

Reply:
After more thinking and discussion, we decided to update the driver version to `2.2.0` as the enhancements to add support for PVC firmware fits under the 'minor fixes and enhancements' so we will increment the y.